lib/models/finamp_models.dart

```diff
@@ -171,7 +172,7 @@ class FinampSettings {
       this.autoloadLastQueueOnStartup = _autoLoadLastQueueOnStartup,
       this.hasCompletedBlurhashImageMigration = true,
       this.hasCompletedBlurhashImageMigrationIdFix = true,
-      this.hasCompleteddownloadsServiceMigration = true,
+      this.hasCompleteddownloadsServiceMigration = false,
```

Comment:
This should probably stay as true by default?

Reply:
yep. didn't intend to change it, at least not after asking about it...